core/DataAccess/RawLogUpdater.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\DataAccess;
+
+use Piwik\Common;
+use Piwik\Db;
+
+class RawLogUpdater
```

Comment:
Wondering why you split RawLogFetcher and RawLogUpdater into two Classes and not into one? It looks like they belong together unless there is a good reason for it.